assets/scripts/evolution/div_table.js

```diff
@@ -0,0 +1,1158 @@
+      "paid_amount": "$2146.70"
+    },
+    {
+      "id": 2,
+      "name": "Anthony Gibson",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "9%",
+      "contract_end_day": "2/21/2017",
+      "total_deals": 99,
+      "payment_cycle": "quarterly",
+      "outstanding_balance": "$14609.61",
+      "paid_amount": "$18079.02"
+    },
+    {
+      "id": 3,
+      "name": "Victor Simmons",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "9%",
+      "contract_end_day": "9/2/2016",
+      "total_deals": 97,
+      "payment_cycle": "monthly",
+      "outstanding_balance": "$3344.06",
+      "paid_amount": "$18358.86"
+    },
+    {
+      "id": 4,
+      "name": "Donna Reyes",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "90%",
+      "contract_end_day": "8/3/2016",
+      "total_deals": 51,
+      "payment_cycle": "monthly",
+      "outstanding_balance": "$8599.40",
+      "paid_amount": "$8154.39"
+    },
+    {
+      "id": 5,
+      "name": "Todd Peters",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "21%",
+      "contract_end_day": "2/11/2017",
+      "total_deals": 92,
+      "payment_cycle": "quarterly",
+      "outstanding_balance": "$10940.73",
+      "paid_amount": "$17910.47"
+    },
+    {
+      "id": 6,
+      "name": "Patricia Carr",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "21%",
+      "contract_end_day": "2/11/2017",
+      "total_deals": 72,
+      "payment_cycle": "annually",
+      "outstanding_balance": "$6385.47",
+      "paid_amount": "$17310.78"
+    },
+    {
+      "id": 7,
+      "name": "Jacqueline Davis",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "23%",
+      "contract_end_day": "11/29/2016",
+      "total_deals": 81,
+      "payment_cycle": "annually",
+      "outstanding_balance": "$5433.61",
+      "paid_amount": "$4281.15"
+    },
+    {
+      "id": 8,
+      "name": "Heather Mcdonald",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "3%",
+      "contract_end_day": "7/18/2016",
+      "total_deals": 98,
+      "payment_cycle": "quarterly",
+      "outstanding_balance": "$10515.56",
+      "paid_amount": "$1112.12"
+    },
+    {
+      "id": 9,
+      "name": "Nicole Willis",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "90%",
+      "contract_end_day": "9/21/2016",
+      "total_deals": 63,
+      "payment_cycle": "annually",
+      "outstanding_balance": "$1014.84",
+      "paid_amount": "$12243.56"
+    },
+    {
+      "id": 10,
+      "name": "Anthony Mills",
+      "img_url": "http://i.imgur.com/7dyjRJe.png",
+      "commission_rate": "21%",
+      "contract_end_day": "9/7/2016",
+      "total_deals": 2,
+      "payment_cycle": "annually",
+      "outstanding_balance": "$10858.51",
+      "paid_amount": "$9096.66"
+    },
+    // {
```

Comment:
@jmatembu Hi I have cleaned the code base. Please review the .js file again and let me know if you have further concerns. :)